src/main/java/teammates/common/util/Const.java

```diff
@@ -767,7 +767,7 @@ public static final String getjQueryUiFilePath(boolean isDevServer) {
         public static final String STUDENT_SHORT_NAME = "studentshortname";
         public static final String STUDENT_PROFILE_EMAIL = "studentprofileemail";
         public static final String STUDENT_PROFILE_INSTITUTION = "studentprofileinstitute";
-        public static final String STUDENT_NATIONALITY = "studentnationality";
+        public static final String STUDENT_NATIONALITY = "studentNationality";
```

Comment:
Yes, because the naming convention in the `StudentProfilePage.java` defines the `id` like this.
And I use it in the `studentProfileDiv.tag` both for the `id` and the `name` of the html element.